doc/plugin_server_upstreamauthority_vault.md

```diff
@@ -23,6 +23,16 @@ The plugin supports **Client Certificate**, **Token** and **AppRole** authentica
 - **Token** method authenticates to Vault using the token in a HTTP Request header. 
 - **AppRole** method authenticates to Vault using a RoleID and SecretID that are issued from Vault.
 
+The plugin must have at least the following permissions(Vault Policy).
```

Comment:
This sentence is somehow confusing for me. Instead of referring to the "plugin permissions", I would rather refer to the capabilities required in the policy. Which is actually pointed below.
We may say something along the lines of 
`The configured token needs to be attached to a policy that has at least the following capabilities: ...`
What do you think?